### ISSUE TITLE
Designer adds/removes whitespaces in code file needlessly

### ISSUE THREAD

**Issue by mocakturk** [dotnet/winforms#9625]
### Environment

Microsoft Visual Studio Professional 2022 (64-bit) - Version 17.6.2

### .NET version

.NET 7

### Did this work in a previous version of Visual Studio and/or previous .NET release?

There was no problem when I was using Visual Studio 2017. However, Resharper was available with VS2017.

### Issue description

I was using VS2017 previously with Resharper. I started to use VS2022 without Resharper now...
I noticed that, whenever I perform a modification on designer, it adds/removes whitespace characters on several lines in the code file. Some examples are below. They are needless. It causes a modification in code file eventhough I didn't. I revert them one by one...

Examples:
![image](https://github.com/dotnet/winforms/assets/106665359/d9fb634c-7524-46d0-bf43-30a236f813c2)

![image](https://github.com/dotnet/winforms/assets/106665359/de5f0345-0f83-45e6-99b0-4ef8f303c3aa)

### Steps to reproduce

- Create a form. Fill the Load event code similar to below. Be careful with the whitespace usage between **(double)** and **x**.
```
private void Form1_Load(object sender, EventArgs e)
{
      int x = 4;
      var y = (double) x;
}
```
- Open designer, do some modification
- Return back to code. Notice that, whitespace is removed. Designer modified the code.

**Comment by elachlan** [comment]
Related #7653

@KlausLoeffelmann you will be interested in this one.

**Comment by KlausLoeffelmann** [comment]
Fixed in 17.8 Preview 1.
We're considering bringing it back to 17.6, but there is no definitive decision yet.

**Comment by merriemcgaw** [comment]
Closing this issue since we're shipping the fix in 17.8 Preview 1. Please feel free to let us know if after Preview 1 you find other inconsistencies in the serialization you want to bring up to our attention.
